addons/l10n_ae/demo/demo_company.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="partner_demo_company_ae" model="res.partner">
+        <field name="name">AE Company</field>
+        
+        <field name="street">Street 123</field>
+        <field name="city">City</field>
+        <field name="country_id" ref="base.ae"/>
+        <field name="state_id" ref="base.state_ae_uq"/>
+        <field name="zip">2000</field>
+        <field name="phone">+1 555 123 8069</field>
```

Comment:
Is there a way to use a bit realistic data?  We could use Odoo Dubai ?

Reply:
The phone number is now well formatted, I will see if I find something not too complicated to generate more realistic addresses, but I don't think its really necessary. Maybe I can translate the Street name, and find the correct zip code format ?